torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -225,11 +235,13 @@ def resize_image(
         elif image.device.type == "cpu":
             # uint8 dtype support for bilinear and bicubic is limited to cpu and
             # according to our benchmarks, non-AVX CPUs should still prefer u8->f32->interpolate->u8 path for bilinear
-            if (interpolation == InterpolationMode.BILINEAR and "AVX2" in torch.backends.cpu.get_cpu_capability()) or (
+            # For torch.compile we use uint8 input and let decomposition work
+            if (interpolation == InterpolationMode.BILINEAR and _can_add_uint8()) or (
```

Comment:
Can we include this entire `if`'s conditions into the `_can_add_uint8()` helper? I.e. something like:

```py

def _can_add_uint8():
	if interpolation == InterpolationMode.BILINEAR:
	    if torch._dynamo.is_compiling():
	        return True
	    else:
	        return "AVX2" in torch.backends.cpu.get_cpu_capability()
        else:
            return interpolation == InterpolationMode.BICUBIC
```

This would make the name `_can_add_unit8()` more logical IMO.


Also, we seem to have some duplicated comments now (those in line 236 for example).